### PR TITLE
Refs #24 Return max Member.hours_owed at 8 hours;

### PR DESCRIPTION
**Pull request by stratigos**
For issue #24.

### CODE REVIEW

File: app/models/member.rb [hunk 1]
```diff
@@ -148,8 +148,10 @@ def work_in(type = :weeks)
     end.floor
   end
 
+  # Members should not go into debt beyond 8 hours, so regardless of status
+  #  or work history, members should never owe more than 8 hours.
   def hours_owed
-    work_in(:months) * monthly_hours
+    (hrs = work_in(:months) * monthly_hours) < 8.0 ? hrs : 8
```
Comment (flah00) [inline app/models/member.rb]: I think it might be more legible if it were written as 
```ruby
hrs = work_in(:months) * monthly_hours
[hrs, 8].min
```

**Comment by stratigos** [comment]
@flah00 Totally agree, I wasn't comfortable with that line, but just hadn't thought of Enumerable's min() when writing that. I'll make the change later tonight, thanks!

**Comment by flah00** [comment]
:+1:

**Comment by flah00** [comment]
Todd, is it just me, or did this commit reverse your styling commit for notes? I'm looking at the current website, ie http://db.ov.bushwickfoodcoop.org/members/16/edit and the note addition is still a checkbox.

**Comment by stratigos** [comment]
@flah00 I just checked locally, and I am still seeing the button for "Note" that I added.

This is on a freshly rebased 'master' branch with no local changes.

Maybe your browser cached the old version of the page, or the push to production diddnt go through correctly? Grasping at straws cuz I dont know :wink:

**Comment by flah00** [comment]
I see what happened. I forgot to push to the ov remote. My bad, sorry for the false alarm.

**Comment by flah00** [comment]
@stratigos 

Looks like I'm going to have to revert this commit. Nick told me after I pushed this change almost everyone who owed hours now has surplus hours (banking hours, in coop parlance). 

Can you look into this and submit another PR?

Thanks!

**Comment by stratigos** [comment]
Sure thing, Ill try and make time this weekend to figure that out. Ill look into some tests for hours_owed vs "hours_banked" too.

**Pull request closed without merging**